boards/realtek/rts5912_evb/rts5912_evb_defconfig

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+# Enable RTS5912 image tool
+CONFIG_REALTEK_RTS5912_BOOTROM_HEADER=y
+CONFIG_REALTEK_RTS5912_HEADER_COMBINE=y
+
+# Serial Driver
+CONFIG_SERIAL=y
+
+# Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Clock support
+CONFIG_CLOCK_CONTROL=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable pin controller
+CONFIG_PINCTRL=y
```

Comment:
-1, drivers need to select this as needed, same for clk control

Reply:
Now, We moved tokens of PINCTRL and CLOCK_CONTROL to serial Kconfig.realtek_rts5912 to select them.

Reply:
@gmarull Can you review this again?